ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -1408,15 +1409,26 @@ smUnused maxClockSkew chunkInfo =
       maxClockSkew
 
 prop_sequential :: MaxClockSkew -> SmallChunkInfo -> Property
-prop_sequential maxClockSkew (SmallChunkInfo chunkInfo) =
+prop_sequential maxClockSkew (SmallChunkInfo chunkInfo) = withMaxSuccess 100 $
```

Comment:
Yeah, indeed. That number will end up being something larger, which we need to determine. My suggestion 100K based on my observations on this bug's reproducibility. This will increase the tests running time by 3 minutes which I think is ok. We could also make use of the nightly builds to run a higher number of test cases.